content/bn/_index.md

```diff
@@ -10,6 +10,33 @@ status: Completed
 ## অবদান
 
 ক্লাউড নেটিভ শব্দকোষে পরিবর্তন, সংযোজন এবং উন্নতির পরামর্শ দেওয়ার জন্য সবাইকে আমন্ত্রণ জানানো হয়েছে। আমরা এই ভাগ করা অভিধানের বিকাশ এবং উন্নতির জন্য CNCF দ্বারা পরিচালিত একটি সম্প্রদায়-চালিত প্রক্রিয়া নিযুক্ত করি। এই শব্দকোষটি ক্লাউড নেটিভ প্রযুক্তির আশেপাশে একটি ভাগ করা শব্দভাণ্ডার সংগঠিত করার জন্য একটি বিক্রেতা-নিরপেক্ষ প্ল্যাটফর্ম প্রদান করে৷ প্রজেক্টের উদ্দেশ্য এবং চার্টার মেনে চলা সকল অংশগ্রহণকারীদের থেকে অবদানকে স্বাগত জানানো হয়।
+---
+title: "ক্লাউড নেটিভ শব্দকোষ"
+status: Completed
+---
```

Comment:
Hello @Zubiii333, those lines created an unexpected UI. Those should be at the start of the file at line 1.

![image](https://github.com/cncf/glossary/assets/60180521/61735136-11c0-4434-bf55-bd7a3c5c4132)

Reply:
Hi @sajibAdhi ! Sorry for the issue created there. I believe its fixed now.